src/utils/gaa.js

```diff
@@ -870,14 +873,24 @@ export class GaaMeteringRegwall {
     self.addEventListener('message', (e) => {
       if (
         e.data.stamp === POST_MESSAGE_STAMP &&
-        e.data.command === POST_MESSAGE_COMMAND_BUTTON_CLICK
+        e.data.command === POST_MESSAGE_COMMAND_GSI_BUTTON_CLICK
       ) {
         // Log button click event.
         logEvent({
           analyticsEvent: AnalyticsEvent.ACTION_SHOWCASE_REGWALL_GSI_CLICK,
           isFromUserAction: true,
         });
       }
+      if (
+        e.data.stamp === POST_MESSAGE_STAMP &&
+        e.data.command === POST_MESSAGE_COMMAND_SIWG_BUTTON_CLICK
+      ) {
+        // Log button click event.
+        logEvent({
+          analyticsEvent: AnalyticsEvent.ACTION_SHOWCASE_REGWALL_SWIG_CLICK,
```

Comment:
Looks like this PR logs this event in two places. This line isn't covered by tests though  (https://app.codecov.io/gh/subscriptions-project/swg-js/pull/2460). Just checking, does the event need to be logged here?

Reply:
Thanks for explaining, that makes sense